spec/cypress/integration/accessibility.js

```diff
@@ -34,9 +34,21 @@ describe("Accessibility", () => {
            }}`
         ).then((lessonParts) => {
           const skip = [
-            "/edt/year-1/summer-1/topic-5/part-3", // $I messing up header ids
-            "/edt/year-2/autumn-1/topic-3/part-3", // $CTA messing up header ids
-            "/teach-first/year-1/spring-2/topic-4/part-2", // $I messing up header ids
+            "/edt/year-1/autumn-1/topic-7/part-1", // header order error
+            "/edt/year-1/autumn-1/topic-8/part-3", // header order error
+            "/edt/year-1/spring-1/topic-7/part-3", // header order error
+            "/edt/year-1/spring-2/topic-7/part-3", // header order error
+            "/edt/year-1/summer-1/topic-5/part-3", // header order error
+            "/edt/year-1/summer-1/topic-8/part-3", // header order error
+            "/edt/year-2/autumn-1/topic-3/part-3", // header order error
+            "/edt/year-1/spring-1/topic-2/part-3", // header order error
+            "/edt/year-1/autumn-1/topic-5/part-3", // header order error
+            "/edt/year-1/autumn-2/topic-2/part-3", // header order error
+            "/teach-first/year-1/spring-2/topic-4/part-2", // header order error
+            "/teach-first/year-1/spring-1/topic-3/part-2", // header order error
+            "/teach-first/year-1/summer-1/topic-3/part-2", // header order error
+            "/ucl/year-1/spring-2/topic-2/part-3", // header order error
+            "/ucl/year-1/summer-1/topic-2/part-3", // header order error
```

Comment:
Try fetching latest content - we have done some accessibility work on it. Sorry I haven't thought about telling you before :)

Reply:
no problem :) , i'll have a go with the latest seed dump from staging